app/assets/stylesheets/scaffolds.scss

```diff
@@ -1,84 +0,0 @@
-body {
```

Comment:
I was wondering about those. I thought maybe they were going to be used for something in the future, so I erred on the side of caution.